runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -11731,6 +11750,9 @@ executeBreakpointedBytecode: {
 #endif
 		, _objectAllocate(currentThread)
 		, _objectAccessBarrier(currentThread)
+if defined(J9VM_OPT_CRIU_SUPPORT)
+		, _dontSkipHooks(isCRIUSupportEnabled(currentThread))
+#endif
```

Comment:
Please add comments `#endif /* DO_HOOKS */` and `#endif /* !defined(J9VM_OPT_CRIU_SUPPORT) */` for this and others.